test/models/user_test.rb

```diff
@@ -1,7 +1,68 @@
 require 'test_helper'
```

Comment:
Pls add association tests for the user. Please use shoulda-gem as I shared in the friendship test comment